macros/src/test/scala/code/chymyst/jc/MacrosSpec.scala

```diff
@@ -230,7 +230,7 @@ class MacrosSpec extends FlatSpec with Matchers with BeforeAndAfterEach {
     // reaction contains all kinds of pattern-matching constructions, blocking molecule in a guard, and unit values in molecules
     val result = go {
     // This generates a compiler warning "class M expects 2 patterns to hold (Int, Option[Int]) but crushing into 2-tuple to fit single pattern (SI-6675)".
-    // Ignore this warning - this case is what we are testing right now, among other cases, so we cannot remove this warning.
+    // However, this crushing is among the issues covered by this test, and we cannot tell scalac to ignore this warning.
```

Comment:
btw, I found where this comment of compiler comes from:
http://alvinalexander.com/java/jwarehouse/scala-2.11/compiler/scala/tools/nsc/transform/patmat/ScalacPatternExpanders.scala.shtml